test/SlowTests/Server/RecordingTransactionOperationsMergerTests.cs

```diff
@@ -0,0 +1,500 @@
+        {
+            var dateString = DateTime.Now.ToString("G").Replace(" ", "").Replace(":", "").Replace(@"/", "");
+            var filePath = $@"C:\Records\record_{dateString}.json";
+
+            var expectedNames = new[] { "Avi" };//}, "Baruch", "Charlotte", "Dylan", "Eli", "Fabia", "George", "Harper" };
+            var users = expectedNames.Select(n => new User { Name = n }).ToArray();
+            User[] documents = { };
+
+
+            //Recording
+            using (var store = GetDocumentStore())
+            {
+                store.Commands().Execute(new StartTransactionsRecordingCommand(filePath));
+                using (var session = store.OpenSession())
+                {
+                    foreach (var user in users)
+                    {
+                        session.Store(user);
+                    }
+
+                    session.SaveChanges();
+                }
+                store.Commands().Execute(new StopTransactionsRecordingCommand());
+                //WaitForUserToContinueTheTest(store);
+            }
+        }
+
+        [Fact]
+        public void JustReplay()
+        {
+            const string filePath = @"C:\Records\test.json";
+
+            using (var store = GetDocumentStore())
+            {
+                store.Commands().Execute(new ReplayTransactionsRecordingCommand(filePath));
+                WaitForUserToContinueTheTest(store);
+            }
+        }
+
+        [Fact]
+        public void Attachments()
+        {
+            var dateString = DateTime.Now.ToString("G").Replace(" ", "").Replace(":", "").Replace(@"/", "");
+            var filePath = $@"C:\Records\record_{dateString}.json";
+
+            var album = new Album
+            {
+                Name = "Holidays",
+                Description = "Holidays travel pictures of the all family",
+                Tags = new[] { "Holidays Travel", "All Family" },
+            };
+
+            using (var store = GetDocumentStore())
+            {
+                store.Commands().Execute(new StartTransactionsRecordingCommand(filePath));
+
+                using (var session = store.OpenSession())
+                using (var picture = File.Open(@"C:\Records\Igal.jpg", FileMode.Open))
+                {
+                    session.Store(album, "albums/1");
+
+                    session.Advanced.Attachments.Store("albums/1", "Igal.jpg", picture, "image/jpeg");
+
+                    session.SaveChanges();
+
+                }
+                store.Commands().Execute(new StopTransactionsRecordingCommand());
+            }
+
+            using (var store = GetDocumentStore())
+            {
+                store.Commands().Execute(new ReplayTransactionsRecordingCommand(filePath));
+                WaitForUserToContinueTheTest(store);
+            }
+        }
+
+        [Fact]
+        public void StartRecordingWithoutStop_ShouldResultInLegalJson()
+        {
+            var filePath = Path.GetTempFileName();
```

Comment:
Better to use `NewDataPath`. Those are deleted after the test.

Reply:
done